tests/Aspire.Hosting.Tests/DistributedApplicationTests.cs

```diff
@@ -423,6 +423,38 @@ public async Task VerifyDockerAppWorks()
         await app.StopAsync();
     }
 
+    [LocalOnlyFact]
+    public async Task KafkaComponentCanProduceAndConsume()
```

Comment:
Checkout https://github.com/dotnet/aspire/pull/1217. We are refactoring how these tests are written. Once that PR is merged, you can merge with it and adjust the tests as necessary.

Reply:
done